### DISCUSSION TITLE
Any PenPot Support in the Roadmap?

### DISCUSSION THREAD

**Discussion by HarrisonCreates** [primefaces/community#3257] [Figma UI Kit] [answered]
Hello PrimeTek fans! I've been using PrimeNG everywhere and I absolutely love it! In particular, the new theming system has me very excited!

In the spirit of open source comradery, I was wondering if a similar thing to the Figma UI kit would be developed for PenPot! With all of the new features and plug-in support that PenPot has been upgraded with (especially the assets and color palette stuff), I could see a really nice ecosystem of open source tooling to support the PrimeTek libraries.

Thoughts?

**Comment by cagataycivici** [comment, marked as answer]
No plans for PenPot or any other tool except Figma. Maintaining a Figma version is a full time job already.
